### PR TITLE
Add a gnav PR template

### PR DESCRIPTION
**Pull request by mokimo**
## Description
Adds a new gnav PR template that can be optionally used by adding a query param to the url. `https://github.com/adobecom/milo/compare/main...mokimo:gnav-pr-template?expand=1` 
-> `https://github.com/adobecom/milo/compare/main...mokimo:gnav-pr-template?expand=1&template=gnav.md`

## Related Issue
Resolves: N/A

## Testing instructions
Can't be tested until after this is merged

## Screenshots (if appropriate):
Make our lives easier
Screenshot 2023-09-05 at 13 40 23

## Types of changes

- [x] Chore (Unit test, setup - no prod code)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Test URLs
**Acrobat:**
- Before: https://www.stage.adobe.com/acrobat/online/sign-pdf.html?martech=off
- After: https://www.stage.adobe.com/acrobat/online/sign-pdf.html?martech=off&milolibs=gnav-pr-template--milo--mokimo

**BACOM:**
- Before: https://business.stage.adobe.com/fr/customer-success-stories.html?martech=off
- After: https://business.stage.adobe.com/fr/customer-success-stories.html?martech=off&milolibs=gnav-pr-template--milo--mokimo

**CC:**
- Before: https://main--cc--adobecom.hlx.live/?martech=off
- After: https://main--cc--adobecom.hlx.live/?martech=off&milolibs=gnav-pr-template--milo--mokimo

**Milo:**
- Before: https://main--milo--adobecom.hlx.page/ch_de/drafts/ramuntea/gnav-refactor?martech=off
- After: https://gnav-pr-template--milo--mokimo.hlx.page/ch_de/drafts/ramuntea/gnav-refactor?martech=off

### CODE REVIEW

File: .github/gnav.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Resolves: [MWPW-111111](https://jira.corp.adobe.com/browse/MWPW-111111)
+
+## Testing instructions
+
+## Screenshots (if appropriate):
+
+## Types of changes
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
```
Comment (overmyheadandbody) [inline .github/gnav.md:13]: Not 100% sure we need these, as we could simply use labels 🤷

**Review by overmyheadandbody — APPROVED** [review]
Great idea to have this! ❤️ 

However, I would argue that since Milo tailors to all these consumers, maybe we might want to open a discussion around whether the larger community sees a benefit in having this as standard across the board. Developers opening PRs won't have to manually search for links for different consumers.

**Comment by mokimo** [comment]
Great idea @overmyheadandbody I opened https://github.com/orgs/adobecom/discussions/1214

File: .github/gnav.md (hunk 1 above)
Comment (mokimo) [reply .github/gnav.md:13]: Fine by me to remove it!

File: .github/gnav.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- Before: https://www.stage.adobe.com/acrobat/online/sign-pdf.html?martech=off
+- After: https://www.stage.adobe.com/acrobat/online/sign-pdf.html?martech=off&milolibs=branch--milo--owner
+
+**BACOM:**
+- Before: https://business.stage.adobe.com/fr/customer-success-stories.html?martech=off
+- After: https://business.stage.adobe.com/fr/customer-success-stories.html?martech=off&milolibs=branch--milo--owner
+
+**CC:**
+- Before: https://main--cc--adobecom.hlx.live/?martech=off
+- After: https://main--cc--adobecom.hlx.live/?martech=off&milolibs=branch--milo--owner
```
Comment (narcis-radu) [inline .github/gnav.md:26]: I think we should use `milolibs=<branch>--milo--<owner>`

**Review by narcis-radu — APPROVED** [review]

**Pull request merged**